search-app/docs/api.md

```diff
@@ -2950,6 +2966,125 @@ Content-Length: 292
 }
 ```
 
+### <a name="variable"></a> Variable
```

Comment:
Right. That's way the document is limited here. I want to keep this here, so user can have some idea on what variable is. We should update ingest api doc to document variable concept and various endpoints. @oubiwann should add that in his next PR.